brew/README.md

```diff
@@ -14,6 +14,16 @@ brew update
 brew upgrade
 ```
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/opt/brew
+```
```

Comment:
There's one custom file that I've added, which wasn't in the docs previously: `~/.local/bin/brew-update-service-install`.

So this section should include it:

```text
~/.config/envman/PATH.env
~/.local/opt/brew
~/.local/bin/brew-update-service-install
```

Reply:
Added.